src/pages/myGeotab/addIns/addInStorage.tsx

```diff
@@ -412,100 +550,162 @@ const deletingAddInData: ReactNode = <div className="paragraph">
+            </ol>
+
+            <h2>No LIKE Statement</h2>
+            <p>
+                Currently there is no support for fuzzy string matching.
+            </p>
+
+            <h2>No AND/OR Statements</h2>
+            <p>
+                The <code className="small-code-sample">WHERE</code> clause cannot perform conjunctions or disjunctions.
+            </p>
+
+            <h2>Security Clearance Matters</h2>
+            <p>
+                Security Clearance limitations allow the following API methods:
+            </p>
+            <ol>
+                <li><strong>Administrator, Supervisor, Default User, Drive App User</strong> {`=>`} "Add/Set/Get/Remove"</li>
+                <li><strong>ViewOnly</strong> {`=>`} "Get"</li>
+                <li><strong>Nothing</strong> {`=>`} None</li>
+            </ol>
+
+            <h2>Small vs Large</h2>
+            <p>
+                While it's possible to create a single AddInData object with an array of details, this approach is less scalable. First is contending with the mandatory limit of 10,000 characters. Second is that it can cause unduly large objects to deal with which can be less memory efficient. Third is that if there is an array of entries and you need to remove one, you will have to remove the whole object and add a new one with the updated list of details. In general, we have found it more useful to treat the AddInData as a simple object which there can be many of.
+            </p>
+        </div>
+    </div>
+);
 
-const additionalResources: ReactNode = <div className="paragraph">
-    <ul>
-        <li><a href="https://github.com/Geotab/sdk-addin-samples/tree/master/storage-api-sample">Storage API Add-in Sample</a></li>
-    </ul>
-</div>;
-
-const addInId: ReactNode = <div className="paragraph">
-    <p>An AddInId must be created before the Storage API methods can be used within your Add-In. This encoded GUID is used to register and identify which Add-In some data is associated. AddInId is a mandatory parameter when calling AddInData methods to Add and Get data. This allows each Add-In's data to be isolated from the data used by other Add-Ins. This allows multiple solutions to each have their own collection of AddInData objects in the same database without the collections mixing. To generate your own AddInId, please use the following <a href="https://geotab.github.io/sdk/software/api/runner.html#sample:generate-addin-guid">example</a>.</p>
-</div>;
+const resources: ReactNode = (
+    <div className="paragraph">
+        <ul>
+            <li>
+                <a href="https://github.com/Geotab/sdk-addin-samples/tree/master/storage-api-sample"
+                    target="_blank"
+                    rel="noreferrer"
+                    aria-label="MyGeotab Storage API Add-In Sample">
+                    Storage API Add-in Sample
+                </a>
+            </li>
+        </ul>
```

Comment:
Don't know if this needs to be a list? We can probably just list the `<a>` tag inside of a `<p>` tag instead.